### PR TITLE
[RC] Migrated UI items for iTwinJS 4.x upgrade for Report Config widget

### PR DESCRIPTION
**Pull request by smmr-dn**
Upgraded CSS variables and replaced deprecated packages/components. Matched UI styling based on G&M widget.

### CODE REVIEW

**Comment by Guillar1** [comment]
Why was pnpm-lock removed?

**Comment by smmr-dn** [comment]
> Why was pnpm-lock removed?

I think I mistook it for the one from the cospace when I tried to mess around to figure out the rush errors before pushing. Let me try committing it again.

File: packages/itwin/reports-config-widget/package.json [hunk 1]
```diff
@@ -99,7 +99,7 @@
     "@typescript-eslint/eslint-plugin": "^5.3.1",
     "@typescript-eslint/parser": "^5.10.1",
     "chai": "^4.2.0",
-    "cpx2": "^3.0.0",
+    "cpx2": "^4.0.0",
```
Comment (Guillar1) [inline packages/itwin/reports-config-widget/package.json:102]: Might have been an accidental version bump here.

Comment (Guillar1) [reply packages/itwin/reports-config-widget/package.json]: Lockfile has to reflect the change as well.

Comment (smmr-dn) [reply packages/itwin/reports-config-widget/package.json]: I updated the lockfile.

File: common/changes/@itwin/reports-config-widget-react/uyen-report-config-migration_2023-07-26-23-02.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/reports-config-widget-react",
+      "comment": "migrated UI items for iTwinJS 4.x upgrade",
```
Comment (Guillar1) [inline common/changes/@itwin/reports-config-widget-react/uyen-report-config-migration_2023-07-26-23-02.json:5]:
```suggestion
      "comment": "Migrated UI items for iTwin UI 2.x and iTwinJS 4.x upgrade.",
```

File: common/changes/@itwin/reports-config-widget-react/uyen-report-config-migration_2023-07-26-23-02.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/reports-config-widget-react",
+      "comment": "Migrated UI items for iTwin UI 2.x and iTwinJS 4.x upgrade.",
+      "type": "patch"
```
Comment (arnobmallickbsw) [inline common/changes/@itwin/reports-config-widget-react/uyen-report-config-migration_2023-07-26-23-02.json:6]: this should be a 'minor' type change
'patch' implies there are no breaking changes, but we have a major dependency overhaul

Comment (smmr-dn) [reply common/changes/@itwin/reports-config-widget-react/uyen-report-config-migration_2023-07-26-23-02.json]: I just changed it to 'minor'.

**Comment by Guillar1** [comment]
Can you give the class `rcw-horizontal-tile-container` `height` into a `min-height`? To keep the sizing consistent even if there is no description.

**Comment by Guillar1** [comment]
image
Can you make this copy icon not borderless? It doesn't look good here anymore.

**Review by Guillar1 — APPROVED** [review]

**Comment by Guillar1** [comment]
@aruniverse

**Review by aruniverse — APPROVED** [review]
pkg.json changes look fine

**Pull request merged**